Mathlib/Data/IsROrC/Basic.lean

```diff
@@ -822,12 +827,34 @@ noncomputable instance Real.isROrC : IsROrC ℝ where
   norm_sq_eq_def_ax z := by simp only [sq, Real.norm_eq_abs, ← abs_mul, abs_mul_self z, add_zero,
     mul_zero, AddMonoidHom.zero_apply, AddMonoidHom.id_apply]
   mul_im_I_ax z := by simp only [MulZeroClass.mul_zero, AddMonoidHom.zero_apply]
+  le_iff_re_im := (and_iff_left rfl).symm
 #align real.is_R_or_C Real.isROrC
 
 end Instances
 
 namespace IsROrC
 
+open scoped ComplexOrder in
+/-- Note this is only an instance with `open scoped ComplexOrder`. -/
```

Comment:
I think it would be nice to keep this portion of the docstring from the `ℂ` instance:
```suggestion
/-- With `z ≤ w` iff `w - z` is real and nonnegative, `IsROrC K` is a star ordered ring.
(That is, a star ring in which the nonnegative elements are those of the form `star z * z`.)

Note this is only an instance with `open scoped ComplexOrder`. -/
```